src/stdx/prng.zig

```diff
@@ -189,26 +189,26 @@ test fill {
     ).diff_fmt("{d}", .{distribution});
 }
 
-/// Genrate an unbiased, uniformly distributed integer r such that 0 ≤ r ≤ max.
+/// Generate an unbiased, uniformly distributed integer r such that 0 ≤ r ≤ max.
 ///
 /// No biased version is provided --- while biased generation is simpler&faster, the bias can be
 /// quite high depending on max!
-pub fn int_inclusive(prng: *PRNG, T: anytype, max: T) T {
-    comptime assert(@typeInfo(T).Int.signedness == .unsigned);
-    if (max == std.math.maxInt(T)) {
-        return prng.int(T);
+pub fn int_inclusive(prng: *PRNG, Int: anytype, max: Int) Int {
```

Comment:
I am confused whether I should approvingly pat myself on the back for noticing a nice pattern elsewhere, and applying it to my code, or scold myself because "elsewhere" was written by past matklad, and apparently I've forgotten the pattern since!